vendor/assets/stylesheets/dvl/core/typography.scss

```diff
@@ -6,18 +6,19 @@
    http://modularscale.com/scale/?px1=12&px2=20&ra1=1.333&ra2=0
 \* ================================================================== */
 
-html { font-size: 75%; } // 12px
-@media only screen and (min-width: 320px) { html { font-size: $maxFontSize * 0.8125; } } // 13px
```

Comment:
Alright, after a bit of research, I'm fine with this, as long as we test our mobile layouts thoroughly. 

Although with the way it is now, is there any functional difference between `rem`s and `px`s? If we need to stick with `rem`s because switching to pixels would be too big of a pain in the butt, that's fine too.